### PR TITLE
Implement WPM and probabilistic sampling for agent responses; also rename mediators to agents.

### LINKED ISSUE #347: Rename "Mediator" to "Agent"
Currently, agents are referenced as "Mediators" throughout the codebase. This may be confusing for experimenters wanting to use agents for simulation purposes.

### PR DESCRIPTION
**Pull request by cjqian**
Fixes #347

### CODE REVIEW

File: utils/src/stages/chat_stage.ts [hunk 1]
```diff
@@ -104,6 +104,7 @@ export interface MediatorConfig {
   name: string;
   avatar: string; // emoji avatar for mediator
   prompt: string;
+  wordsPerMinute: number; // Typing speed
```
Comment (vivtsai) [inline utils/src/stages/chat_stage.ts:107]: Don't forget to update experiment version!

**Review by vivtsai — CHANGES_REQUESTED** [review]

File: frontend/src/components/stages/chat_editor.scss [hunk 2]
```diff
@@ -108,4 +108,9 @@ pr-textarea {
 .tab {
   margin-top: 0px;
   margin-left: calc(48px + common.$spacing-medium);
+}
+
+.error-message {
+  font-style: italic;;
```
Comment (vivtsai) [inline frontend/src/components/stages/chat_editor.scss:114]: double semicolon?

Comment (cjqian) [reply frontend/src/components/stages/chat_editor.scss:114]: Fixed, thank you!

**Comment by cjqian** [comment]
@mkbehr as FYI

**Review by vivtsai — APPROVED** [review]

**Pull request merged**